src/main/java/com/imsweb/validation/functions/MetafileContextFunctions.java

```diff
@@ -975,6 +975,88 @@ else if (index instanceof Set)
 
     /**
      * Special genedit method. Internal use only.
+     */
+    public boolean GEN_SQLLOOKUP(List<List<Object>> table, List<Object> columnNames, Object value) {
+        if (columnNames.isEmpty())
+            return true;
+
+        String val = GEN_TO_STRING(value);
+
+        int[] colNumbers = new int[columnNames.size()];
+        for (int i = 0; i < colNumbers.length; i++) {
+            colNumbers[i] = table.get(0).indexOf(columnNames.get(i));
+            if (colNumbers[i] == -1)
+                return true;
+        }
+
+        for (int r = 1; r < table.size(); r++) {
+            int index = 0;
+            boolean match = false;
```

Comment:
Yep. I changed the guts of that `for`-loop to match the other method, but this one doesn't need those variables.